src/Sepes.Infrastructure/Service/LookupService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Sepes.Infrastructure.Dto;
+using System.Collections.Generic;
+
+namespace Sepes.Infrastructure.Service
+{
+    public class LookupService : ILookupService
+    {
+        public IEnumerable<LookupDto> GetAzureRegions()
+        {
+            return new List<LookupDto>()
+            {
+                new LookupDto{ Key= "NorwayEast", DisplayValue = "Norway East" },
+                new LookupDto{  Key= "NorthEurope",  DisplayValue = "North Europe" },
+                new LookupDto{Key= "WestEurope",  DisplayValue = "West Europe" },
+                new LookupDto{ Key= "NorwayWest",  DisplayValue =  "Norway West" }
```

Comment:
Er det en foreløpig løsning eller var det greit sånn?

Reply:
Foreløpig, bare for å ha noe mat du kan hente ut